index.ts

```diff
@@ -0,0 +1,72 @@
+import express, { Express, Request, Response } from "express";
+import bodyParser from "body-parser";
+import { auctionPaid } from "./types/interfaces";
+import { submitPayment } from "./submitPayment";
+import { saveReciept } from "./saveReceipt";
+import { searchPaid } from "./searchPaid";
+import cors from "cors";
+
+const app: Express = express();
+app.use(bodyParser.json());
+app.use(cors());
+app.use(
+    bodyParser.urlencoded({
+        extended: true,
+    })
+);
+
+const port = 3000;
+
+app.get('/', (req: Request, res: Response) => {
+
+});
+//pay and save reciept
+app.post(
+    "/submitPayment",
+    async(
+        req: Request<unknown, unknown, auctionPaid, unknown>,
+        res: Response
+    ) =>{
+        try{
+
+            const response = await submitPayment();
+            res.send(response);
```

Comment:
if you do this, the `saveReceipt()` never runs I believe, since the response will be sent here, just check if the response is true and call below